website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

```diff
@@ -35,14 +48,69 @@ annotation `consul.hashicorp.com/auto-join-port` to an integer value or
 a named port to specify the port for the auto-join to return. This enables
 different pods to have different exposed ports.
 
-## Networking
+### Auto-join on the Pod network
+In the default Consul Helm chart installation, the Consul clients and servers
+are deployed as pods and routeable only via their `podIP:containerPort` for
+gossip/RPC. This would mean any external client agents joining would need to
+be able to have connectivity to those pod IPs.
+
+Given you have the [official Helm chart](/docs/k8s/helm) installed with the default values, do the following to join an external client agent. 
+
+1. Make sure the pod IPs of the clients and servers in Kubernetes are routeable from the VM and that the VM IP (`$ADVERTISE_IP`) is routeable from the pods, adding any firewall rules necessary to allow traffic on the gossip and RPC ports.
 
-Consul typically requires a fully connected network.
-Because the Consul Helm chart currently doesn't allow exposing servers' gossip ports via a `hostPort`,
-nodes outside of Kubernetes joining a cluster running within Kubernetes must be able to communicate
-to pod IPs via the network. Note that the auto-join provider discussed above will use pod IPs by default.
+2. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config`.
+
+2.  On the external VM, run: 
+```shell-session
+consul agent \
+  -advertise="$ADVERTISE_IP" \
+  -retry-join='provider=k8s label_selector="app=consul,component=server"'
+  -bind=0.0.0.0 \
+  -client=0.0.0.0 \
+  -hcl='leave_on_terminate = true' \
```

Comment:
I think we can omit it here because it's true by default for clients. We are setting it explicitly in the helm chart to support earlier versions of consul that defaulted to false, but I don't think we need it for these docs.